admin-dev/themes/default/template/controllers/orders/_customized_data.tpl

```diff
@@ -221,7 +221,7 @@
 								<input onchange="checkPartialRefundProductAmount(this)" type="text" name="partialRefundProduct[{$product['id_order_detail']|intval}]" />
 								{if !$currencySymbolBeforeAmount}<div class="input-group-addon">{$currency->sign}</div>{/if}
 							</div>
-							<p class="help-block"><i class="icon-warning-sign"></i> {l s='(Max %amount_refundable% %tax_method%)' sprintf=[ '%amount_refundable%' => Tools::displayPrice(Tools::ps_round($amount_refundable, 2), $currency->id), '%tax_method%' => $smarty.capture.TaxMethod] d='Admin.Orderscustomers.Help'}</p>
+							<p class="help-block"><i class="icon-warning-sign"></i> {l s='(Max %amount_refundable% %tax_method%)' sprintf=[ '%amount_refundable%' => Context::getContext()->getCurrentLocale()->formatPrice(Tools::ps_round($amount_refundable, 2), Currency::getCurrencyInstance($currency->id)->iso_code), '%tax_method%' => $smarty.capture.TaxMethod] d='Admin.Orderscustomers.Help'}</p>
```

Comment:
I'm worried about this hard-coded 2, if it's display/compute precision then it should be modified (cf what @matthieu-rolland has done on this subject)
And I'm not even sure it should be rounded here since anyway the CLDR manages the precision when displaying a price now Maybe you could remove this `Tools::ps_round` altogether

Reply:
You can get the display precision of a currency from the context 

`Context::getContext()->currency->precision`

But @jolelievre might be right about CLDR displaying the price, are you talking about a specific method ?

Reply:
ok so the formatPrice already deals the rounding

Reply:
:ok_hand: